pkg/snapshot/snapshot_test.go

```diff
@@ -63,19 +63,24 @@ func TestSnapshotFSFileChange(t *testing.T) {
 		fooPath: "newbaz1",
 		batPath: "baz",
 	}
-	for _, dir := range util.ParentDirectoriesWithoutLeadingSlash(fooPath) {
-		snapshotFiles[dir] = ""
-	}
-	for _, dir := range util.ParentDirectoriesWithoutLeadingSlash(batPath) {
-		snapshotFiles[dir] = ""
-	}
-	numFiles := 0
+
+	// Their parents didn't change so they shouldn't be included
```

Comment:
I would prefer to remove the code, we can always recover it from git.

Reply:
Yes, plan is to remove after testing it finished